src/GlobalHelper.php

```diff
@@ -49,14 +49,19 @@ public static function getInput(): InputInterface
         return self::$input ?? throw new \LogicException('Input not available yet.');
     }
 
-    public static function setOutput(OutputInterface $output): void
```

Comment:
maybe, but imo GlobalHelper is mainly used internally, maybe we should declare it like that instead ?

Reply:
That's what I thought at first but then remembered that we offered some setter to configure the app globally like for the cache, logger or http client, so probably not so internal IMO

Reply:
Maybe have custom function instead of using that then ? This class have both external / internal public methods from what i see, we should avoid that and provide a clean external entrypoint for this behavior, but not the subject of this PR,